src/query_builder.ts

```diff
@@ -9,7 +15,8 @@ export default function interpolateKustoQuery(query: string, scopedVars?: Scoped
 
   query = query.replace(macroRegexp, (match, p1, p2) => {
     if (p1 === 'contains') {
-      return getMultiContains(p2);
+      const replaced = templateSrv.replace(p2, scopedVars, interpolateVariable);
+      return getMultiContains(replaced);
     }
     return match;
   });
```

Comment:
You also need to call `replace` when evaluating `$__escapeMulti` below since that can be called with a template variable as an argument.

Reply:
I've made some changes to this (it was a bit bigger than just putting `replace` in there). Lmk what you think 😊

Reply:
you are right, I was looking at the github view in which that piece of code was collapsed so I didn't see it :sweat_smile: